util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A utility to generate dcid for statistical variables."""
+
+import copy
+import re
+
+# Global constants
+# Regex to match the quantity notations - [value quantity], [quantity value]
+# Example matches: [2 Person], [Person 2]
+_QUANTITY_REGEX_1 = re.compile(
+    r'\[(?P<value>-|-?\d+(\.\d+)?) (?P<quantity>[A-Za-z_/\d]+)\]')
+_QUANTITY_REGEX_2 = re.compile(
+    r'\[(?P<quantity>[A-Za-z_/\d]+) (?P<value>-|-?\d+(\.\d+)?)\]')
+
+# Regex to match the quantity range notations -
+# [lower_limit upper_limit quantity], [quantity lower_limit upper_limit]
+# Example matches: [10000 14999 USDollar], [USDollar 10000 14999]
+_QUANTITY_RANGE_REGEX_1 = re.compile(r'\[(?P<lower_limit>-|-?\d+(\.\d+)?) '
+                                     r'(?P<upper_limit>-|-?\d+(\.\d+)?) '
+                                     r'(?P<quantity>[A-Za-z_/\d]+)\]')
+_QUANTITY_RANGE_REGEX_2 = re.compile(r'\[(?P<quantity>[A-Za-z_/\d]+) '
+                                     r'(?P<lower_limit>-|-?\d+(\.\d+)?) '
+                                     r'(?P<upper_limit>-|-?\d+(\.\d+)?)\]')
+
+# These are the default properties ignored during dcid generation
+_DEFAULT_IGNORE_PROPS = ('unit', 'Node', 'memberOf', 'typeOf',
+                         'constraintProperties', 'name', 'description',
+                         'descriptionUrl', 'label', 'url', 'alternateName')
+
+# Regex to match prefixes to be removed from constraints
+_CONSTRAINT_PREFIX_REGEX = re.compile(r'^(USC|CDC|DAD|BLS|NCES|ACSED)')
+
+
+def _capitalize_process_word(word):
```

Comment:
You can add a return type indicator in the function signature as follows:
def _capitalize_process_word(word) -> str:

Reply:
Done.